src/libraries/System.Private.CoreLib/src/System/Runtime/Intrinsics/Arm/Sve.cs

```diff
@@ -2613,6 +2613,290 @@ public new abstract class Arm64 : AdvSimd.Arm64
         /// </summary>
         public static unsafe Vector<long> SignExtendWideningUpper(Vector<int> value) => SignExtendWideningUpper(value);
 
+
+        ///  Non-truncating store
+
+        /// <summary>
+        /// void svst1[_u8](svbool_t pg, uint8_t *base, svuint8_t data)
+        ///   ST1B Zdata.B, Pg, [Xarray, Xindex]
+        ///   ST1B Zdata.B, Pg, [Xbase, #0, MUL VL]
+        /// </summary>
+        public static unsafe void Store(Vector<byte> mask, byte* address, Vector<byte> data) => Store(mask, address, data);
+
+        /// <summary>
+        /// void svst2[_u8](svbool_t pg, uint8_t *base, svuint8x2_t data)
+        ///   ST2B {Zdata0.B, Zdata1.B}, Pg, [Xarray, Xindex]
+        ///   ST2B {Zdata0.B, Zdata1.B}, Pg, [Xbase, #0, MUL VL]
+        /// </summary>
+        public static unsafe void Store(Vector<byte> mask, byte* address, (Vector<byte> Value1, Vector<byte> Value2) data) => Store(mask, address, data);
+
+        /// <summary>
+        /// void svst3[_u8](svbool_t pg, uint8_t *base, svuint8x3_t data)
+        ///   ST3B {Zdata0.B - Zdata2.B}, Pg, [Xarray, Xindex]
```

Comment:
Please remove `ST* [Xarray, Xindex]` line because we are not generating that with this PR.

Reply:
Done